admin/tabs/general/roles/RoleModel.ts

```diff
@@ -270,6 +272,16 @@ export class RoleModel extends HoistModel {
                 filterable: true
             },
             columns: [
+                ...(this.readonly
+                    ? []
+                    : [
+                          {
+                              ...actionCol,
+                              actionsShowOnHoverOnly: true,
+                              width: calcActionColWidth(3),
+                              actions: [this.editAction(), this.cloneAction(), this.deleteAction()]
```

Comment:
I personally think this is overkill -- if we think double-click editing not obvious, would just add an edit button

Dont't think a delete action or a clone action is warranted.

Reply:
also -- if we do go forward with this PR, we conveniently have an `omit` property on column spec that would make this a lot neater!

Reply:
Thanks for the `omit` reminder.  Applied.

I trimmed the column to just the one action, `edit`.  Delete and Clone I don't think will be used too often, and can eventually be found in the context menu.

But for `edit`, I think its really important to make that more discoverable.
We have a new UI that is going to be used 90% of the time for adding/removing users to roles, and we've hidden the button that gets you there.

Reply:
ok, thanks!